certify-service/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,191 @@
+                      'cryptographic_binding_methods_supported': {'did:jwk'},\
+                      'cryptographic_suites_supported': {'Ed25519Signature2020'},\
+                      'proof_types_supported': {'jwt'},\
+                      'credential_definition': {\
+                          'type': {'VerifiableCredential'},\
+                          'credentialSubject': {\
+                              'fullName': {'display': {{'name': 'Name','locale': 'en'}}}, \
+                              'mobile': {'display': {{'name': 'Phone Number','locale': 'en'}}},\
+                              'dob': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
+                              'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
+                              'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
+                              'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
+                              'policyIssuedOn': {'display': {{'name': 'Policy Issued On','locale': 'en'}}},\
+                              'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
+                              'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
+                              'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
+                       }},\
+                      'display': {{'name': 'Life Insurance Verifiable Credential', \
+                                    'locale': 'en', \
+                                    'logo': {'url': 'https://sunbird.org/images/sunbird-logo-new.png','alt_text': 'a square logo of a Sunbird'},\
+                                    'background_color': '#FDFAF9',\
+                                    'text_color': '#7C4616'}},\
+                       'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'}\
+              }}\
+   }\
+}
+
+## ------------------------------------------- Integrations ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.sunbirdrc.integration
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
+
+##--------------------sunbird registry authentication related demo configuration-------------------------##
+
+mosip.certify.authenticator.sunbird-rc.auth-factor.kba.individual-id-field=policyNumber
+mosip.certify.authenticator.sunbird-rc.auth-factor.kba.field-details={{"id":"policyNumber", "type":"text", "format":""},{"id":"fullName", "type":"text", "format":""},{"id":"dob", "type":"date", "format":"dd/mm/yyyy"}}
+mosip.certify.authenticator.sunbird-rc.auth-factor.kba.registry-search-url=http://localhost:8000/registry/api/v1/Insurance/search
+mosip.certify.authenticator.sunbird-rc.kba.entity-id-field=osid
```

Comment:
Since these properties are related to the authenticator, do we need this in the certify service ?

Reply:
removed these fields